src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -110,6 +110,10 @@ public function copyFile(string $source, string $target, array $options)
 
     private function removeFilesFromDir(string $source, string $target)
     {
+        if (!is_dir($source)) {
+            $this->write(sprintf('  Skipped directory <fg=green>"%s"</>: unable to determine which files to remove', $this->path->relativize($target)));
```

Comment:
I'd suggest removing the message as I don't see what it would be useful for.

Reply:
I removed it. I still think it would be nice to have it at least when running flex with `-vvv`, but that would require even more code changes, so perhaps it's best to keep it simple at this point. From my point of view the use case is developers setting up a private flex recipe. The documentation about doesn't go into too much detail about how flex works and I think that having extra information on why a directory was not removed during unconfiguration would save users from unnecessary confusion (and frustration). Alternatively, the documentation could explain in more detail what unconfiguration does and doesn't do. The naive expectation is that it completely cleans up, but that's not the reality (for good reasons). Either way I don't feel strongly about it.